apps/idos-data-dashboard/src/routes/dashboard/e2e/credential-filtering/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { HStack, Heading, List, ListItem, Stack, Text, VStack } from "@chakra-ui/react";
+import { useQuery } from "@tanstack/react-query";
+
+import { DataError } from "@/components/data-error";
+import { DataLoading } from "@/components/data-loading";
+import { NoData } from "@/components/no-data";
+import { useIdOS } from "@/core/idos";
+
+const useFetchFilteredCredentials = () => {
+  const { sdk } = useIdOS();
+
+  return useQuery({
+    queryKey: ["e2e-credential-filtering"],
+    queryFn: () => sdk.data.listCredentialsFilteredByCountries(["DE"]),
+  });
+};
+
+export function Component() {
+  const credentials = useFetchFilteredCredentials();
```

Comment:
Nit:

```suggestion
  const credential_ids = useFetchFilteredCredentials();
```

Unsure if I care, just because it jumped out at me.